### PR TITLE
upgrade jackson dataformat cbor version to 2.17.0

### PR DESCRIPTION
**Pull request by elhametemad**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -87,7 +87,8 @@ lazy val orchardWS = (project in file("orchard-ws")).
       "com.fasterxml.jackson.core" % "jackson-databind" % "2.15.3",
       "com.fasterxml.jackson.core" % "jackson-core" % "2.15.3",
       "com.fasterxml.jackson.module" %% "jackson-module-scala" % "2.15.3",
-      "com.fasterxml.jackson.dataformat" %% "jackson-dataformat-cbor" % "2.17.0"
+      "com.fasterxml.jackson.dataformat" %% "jackson-dataformat-cbor" % "2.17.0",
+      "io.netty" % "netty-handler" % "5.0.0.Alpha1"
```
Comment (realstraw) [inline build.sbt:91]: alpha? 🤔

Comment (realstraw) [reply build.sbt:91]: would update to play 3.0.5 solve the issue?

Comment (elhametemad) [reply build.sbt:91]: @realstraw I had the same dilema on using alpha version. That is why I removed in my previous commit but returned it back to have conversation on it.

Comment (elhametemad) [reply build.sbt:91]: I will remove this netty upgrade from this PR to have the fix with play upgrade

File: build.sbt [hunk 2]
```diff
@@ -86,7 +86,8 @@ lazy val orchardWS = (project in file("orchard-ws")).
       // the transitive jackson dependencies from play framework on has security vulnerabilities
       "com.fasterxml.jackson.core" % "jackson-databind" % "2.15.3",
       "com.fasterxml.jackson.core" % "jackson-core" % "2.15.3",
-      "com.fasterxml.jackson.module" %% "jackson-module-scala" % "2.15.3"
+      "com.fasterxml.jackson.module" %% "jackson-module-scala" % "2.15.3",
```
Comment (realstraw) [inline build.sbt:89]: let's make the version consistent with the other

**Review by realstraw — APPROVED** [review]

**Pull request merged**